src/App.js

```diff
@@ -1,25 +1,83 @@
-import logo from './logo.svg';
-import './App.css';
+import { useState } from "react";
+import "./App.css";
+import contactsArr from "./contacts.json";
 
 function App() {
+  let fiveContacts = contactsArr.slice(0, 5);
+
+  /** Iteration 3 | Add New Random Contacts **/
+  const [contacts, setContact] = useState(fiveContacts);
+  const remainingContact = contactsArr.slice(6, contactsArr.length);
+  const randomContactIndex = Math.floor(
+    Math.random() * remainingContact.length
+  );
+  const randomContact = remainingContact[randomContactIndex];
+  const randomArr = fiveContacts.concat(randomContact);
```

Comment:
The issue with those five/six contacts should be solved when you do 
`  const randomArr = [...contacts].concat(randomContact);`
Because you always use randomArr and randomArr is always same five contacts (fiveContacts) and not your state. 
If you use a copy of state here instead you can have more that 6 contacts and not always same 5. 
Let me (Joanne) know if you have any questions

Reply:
Also: it would be better if you use setState with callback